jdaviz/core/loaders/importers/image/image.py

```diff
@@ -239,8 +239,12 @@ def __call__(self, show_in_viewer=True):
             else:
                 # If data_label is not a prefix, we use it as is.
                 data_label = base_data_label
+            # For the purposes of the DQ plugin, only the DQ extension can set the
+            # SCI/DATA extension to be it's parent
+            set_parent = (parent if (parent and parent != data_label and ext.lower() == 'dq')
+                          else None)
```

Comment:
doesn't this effectively disable the parent implementation for any case except DQ?

Reply:
As mentioned offline, this needs to be here for now because the DQ extension still requires setting the SCI/DATA extension as it's parent. If we change the parenting code down the line or change how the DQ plugin finds the DQ layer, this can be moved/replaced.

Reply:
I still think it would be worth moving the parenting logic inside the for-loop if that is what is necessary (but I guess can be a small follow-up effort if there isn't time to get it in here)